modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -13,29 +13,10 @@ object UserDao {
     select("root")
 
   def select(id: String): ConnectionIO[User[Nothing]] =
-    sql"""
-      SELECT id, first, last, email, staff
-      FROM gem_user
-      WHERE id = $id
-    """.query[(String, String, String, String, Boolean)]
-       .unique.map { case (i, f, l, e, s) =>
-         User(i, f, l, e, s, Map.empty)
-       }
+    Statements.select(id).unique
```

Comment:
So when you run the query it will not only read the data but also verify that the data types between the tuple in `query` and the db match? that would be neat

Does it do it with JDBC metadata?